lib/src/test/java/vmath/Mat2Test.java

```diff
@@ -22,6 +22,33 @@ void givenIndices_whenGetCalled_thenReturnValueMatches() {
         //TODO
     }
 
+    @Test
+    void givenArgs_whenMat2Created_thenMatrixHasGivenValues() {
+        float[] validValues = { 2.0f, 3.0f, 4.0f, 5.0f };
+
+        Mat2 mat2 = new Mat2(validValues);
+
+        for (int i = 0; i < 2; i++) {
+            for (int j = 0; j < 2; j++) {
+                assertEquals(validValues[i * 2 + j], mat2.get(i, j));
+            }
+        }
+    }
+
+    @Test
+    void givenWrongSize_whenMat2Created_thenThrowsException() {
+
+        float[] invalidSize= { 2.0f, 3.0f, 4.0f};
```

Comment:
Be more careful with the style (whitespaces, empty lines)! Problems are all over the place and not just in the lines mentioned.